content/blog/deploying-mongodb-application-on-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,948 @@
+    signature: {
+      hash: Binary(Buffer.from("0000000000000000000000000000000000000000", "hex"), 0),
+      keyId: Long("0")
+    }
+  },
+  operationTime: Timestamp({ t: 1669297950, i: 1 })
+}
+rs0 [direct: primary] test>
+
+Add mongo-1 pod as Secondary Replica
+
+rs0 [direct: primary] test> rs.add("10.192.3.59:27017")
+{
+  ok: 1,
+  '$clusterTime': {
+    clusterTime: Timestamp({ t: 1669298574, i: 1 }),
+    signature: {
+      hash: Binary(Buffer.from("0000000000000000000000000000000000000000", "hex"), 0),
+      keyId: Long("0")
+    }
+  },
+  operationTime: Timestamp({ t: 1669298574, i: 1 })
+}
+
+Validate that mongo-1 pod is registered as Secondary Replica
+
+rs0 [direct: primary] test> rs.status()
+{
+  set: 'rs0',
+  date: ISODate("2022-11-24T14:03:52.376Z"),
+  myState: 1,
+  term: Long("1"),
+  syncSourceHost: '',
+  syncSourceId: -1,
+  heartbeatIntervalMillis: Long("2000"),
+  majorityVoteCount: 2,
+  writeMajorityCount: 2,
+  votingMembersCount: 2,
+  writableVotingMembersCount: 2,
+  optimes: {
+    lastCommittedOpTime: { ts: Timestamp({ t: 1669298630, i: 1 }), t: Long("1") },
+    lastCommittedWallTime: ISODate("2022-11-24T14:03:50.861Z"),
+    readConcernMajorityOpTime: { ts: Timestamp({ t: 1669298630, i: 1 }), t: Long("1") },
+    appliedOpTime: { ts: Timestamp({ t: 1669298630, i: 1 }), t: Long("1") },
+    durableOpTime: { ts: Timestamp({ t: 1669298630, i: 1 }), t: Long("1") },
+    lastAppliedWallTime: ISODate("2022-11-24T14:03:50.861Z"),
+    lastDurableWallTime: ISODate("2022-11-24T14:03:50.861Z")
+  },
+  lastStableRecoveryTimestamp: Timestamp({ t: 1669298619, i: 4 }),
+  electionCandidateMetrics: {
+    lastElectionReason: 'electionTimeout',
+    lastElectionDate: ISODate("2022-11-24T13:49:50.807Z"),
+    electionTerm: Long("1"),
+    lastCommittedOpTimeAtElection: { ts: Timestamp({ t: 1669297790, i: 1 }), t: Long("-1") },
+    lastSeenOpTimeAtElection: { ts: Timestamp({ t: 1669297790, i: 1 }), t: Long("-1") },
+    numVotesNeeded: 1,
+    priorityAtElection: 1,
+    electionTimeoutMillis: Long("10000"),
+    newTermStartDate: ISODate("2022-11-24T13:49:50.829Z"),
+    wMajorityWriteAvailabilityDate: ISODate("2022-11-24T13:49:50.840Z")
+  },
+  members: [
+    {
+      _id: 0,
+      name: '10.192.3.58:27017',
+      health: 1,
+      state: 1,
+      stateStr: 'PRIMARY',
+      uptime: 4514,
+      optime: { ts: Timestamp({ t: 1669298630, i: 1 }), t: Long("1") },
+      optimeDate: ISODate("2022-11-24T14:03:50.000Z"),
+      lastAppliedWallTime: ISODate("2022-11-24T14:03:50.861Z"),
+      lastDurableWallTime: ISODate("2022-11-24T14:03:50.861Z"),
+      syncSourceHost: '',
+      syncSourceId: -1,
+      infoMessage: '',
+      electionTime: Timestamp({ t: 1669297790, i: 2 }),
+      electionDate: ISODate("2022-11-24T13:49:50.000Z"),
+      configVersion: 4,
+      configTerm: 1,
+      self: true,
+      lastHeartbeatMessage: ''
+    },
+    {
+      _id: 1,
+      name: '10.192.3.59:27017',
```

Comment:
yellow highlighted, and I think I now see why :-)

Reply:
:)